src/Sylius/Bundle/CustomerBundle/Resources/translations/flashes.fr.yml

```
@@ -3,4 +3,4 @@
 
 sylius:
     customer:
-        register: 'Merci pour votre inscription, vous allez recevoir un mail pour vérifier votre compte.'
+        register: 'Merci pour votre enregistrement, vous recevrez un courriel pour vérifier votre compte.'
```

```suggestion
        register: 'Merci pour votre inscription, vous allez recevoir un mail pour vérifier votre compte.'
```
Unnecessary change here as well the original worked better